cypress/integration/percy/page_screenshots.js

```diff
@@ -6,16 +6,26 @@ const pages = [
   { name: 'Group', url: '/groups/1' },
   { name: 'Create Destination - Types', url: '/destinations/new' },
   { name: 'Organization Settings', url: '/settings/organization' },
-  { name: 'User Profile', url: '/users/me' },
 ];
 
 describe('Percy Page Screenshots', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
   pages.forEach((page) => {
     it(`takes a screenshot of ${page.name}`, () => {
-      cy.login();
       cy.visit(page.url);
       cy.wait(1000);
       cy.percySnapshot(page.name);
     });
   });
+
+  it('takes a screenshot of User Profile', () => {
+    cy.visit('/users/me');
+    cy.getByTestId('ApiKey').then(($apiKey) => {
+      $apiKey.val('secret');
```

Comment:
FYI @arikfr 
Revisited for a good reason :slightly_smiling_face:

Reply:
Can this be done with stubs as well? Would be a more standard way of doing this.

Reply:
> Can this be done with stubs as well? Would be a more standard way of doing this.

Stubs were my first try, anything similar to Rspec's `allow(obj).to receive(:message).and_return(:value)`. However, I could not find a way to replace it without stubbing the whole request to the user. Since it would be more complex, I went with this Javascript change.